manifests/install/repos/extra.pp

```diff
@@ -30,12 +30,7 @@
   }
 
   if $ipa_authentication and $configure_ipa_repo {
-    yumrepo { 'adelton-identity':
```

Comment:
I agree with @domcleal: a deprecation means it still works but issues a warning. Removing the functionality means breaking the API and thus a major version bump. Now I don't care if we do break the API in this way, but it is breaking the API.